att_doc_retriever_sample/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+FROM python:3.7.9-alpine3.12
+
+# Copy the Rust compiled binary which retrieves the Attestation document to container
+COPY att_doc_retriever_sample/rs/target/x86_64-unknown-linux-musl/release/att_doc_retriever_sample att_doc_retriever_sample/
```

Comment:
Add the whole build operation to the container instead of requiring a separate build step. You can add a separate stage of the build and copy the result from there: https://docs.docker.com/develop/develop-images/multistage-build/.